meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx8/lmp.cfg

```diff
@@ -26,9 +26,8 @@ CONFIG_HASH=y
 CONFIG_PARTITION_UUIDS=y
 CONFIG_TEE=y
 CONFIG_OPTEE=y
+CONFIG_OPTEE_LIB=y
```

Comment:
why do we need to enable this explicitly, if it's selected automatically:

```
  1 config OPTEE_LIB                                                                
  2         bool "Support OPTEE library"                                            
  3         default y if OPTEE || OPTEE_IMAGE                                       
  4         help                                                                    
  5           Selecting this option will enable the shared OPTEE library code. 
```

Reply:
I believe there is a bug in optee as this option will only show itself when OPTEE_LIB is enabled:

`obj-$(CONFIG_OPTEE_LIB) += optee/`

Reply:
If it was only CONFIG_OPTEE, then it would be selected by default indeed.

Reply:
It was changed to be this way from what it looks like, so until this is properly fixed we will have to explicitly add  CONFIG_OPTEE_LIB.

```
diff --git a/lib/Makefile b/lib/Makefile
index 09e380eb66c..962470f4962 100644
--- a/lib/Makefile
+++ b/lib/Makefile
@@ -16,7 +16,7 @@ obj-$(CONFIG_FIT) += libfdt/
 obj-$(CONFIG_OF_LIVE) += of_live.o
 obj-$(CONFIG_CMD_DHRYSTONE) += dhry/
 obj-$(CONFIG_ARCH_AT91) += at91/
-obj-$(CONFIG_OPTEE) += optee/
+obj-$(CONFIG_OPTEE_LIB) += optee/
 obj-$(CONFIG_ASN1_DECODER) += asn1_decoder.o
 obj-y += crypto/

diff --git a/lib/optee/Kconfig b/lib/optee/Kconfig
```